cli/src/templates/cmakelists.cmake

```diff
@@ -53,6 +53,10 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/tree-sitter-PARSER_NAME.pc"
 install(TARGETS tree-sitter-PARSER_NAME
         LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}")
 
+file(GLOB QUERIES queries/*.scm)
+install(FILES ${QUERIES}
+        DESTINATION "${CMAKE_INSTALL_DATADIR}/tree_sitter/tree-sitter-PARSER_NAME/queries")
```

Comment:
```suggestion
        DESTINATION "${CMAKE_INSTALL_DATADIR}/tree-sitter/queries/PARSER_NAME")
```

Reply:
Works for me.  I admit I vacillated between these two possible arrangements before suggesting the one I originally did.